projectional/src/main/java/jetbrains/jetpad/projectional/cell/BaseProjectionalSynchronizer.java

```diff
@@ -66,6 +70,8 @@
   private DeleteHandler myDeleteHandler = DeleteHandler.EMPTY;
   private ContentKind<SourceItemT> myItemKind;
   private Function<SourceItemT, SourceItemT> myCloner;
+  private ListMap<ContentKind, ContentToItemMapper> myCompatibleContentKinds = new ListMap<>();
+  private ListMap<ContentKind, ContentToItemsListMapper> myContentKindsCompatibleToItemsList = new ListMap<>();
```

Comment:
It means: some content kind which can be mapped to `List<SourceItemT>`. But it doesn't mean it's a list itself, for example, String. That's why `myListContentKind` seems to be not precise.